.github/workflows/e2e-windows.yml

```diff
@@ -137,3 +135,15 @@ jobs:
         shell: pwsh
         run: |
           .\constellation.exe iam destroy --debug -y
+
+      - name: Notify about failure
+        if: |
+          failure() && github.event_name == 'workflow_call'
+        continue-on-error: true
+        uses: ./.github/actions/notify_failure
```

Comment:
Kubernetes version is a required field.

Reply:
Do you mean another field here? I don't think it's actually required

Reply:
true